ds_judgements_public_ui/templates/judgment/detail.html

```diff
@@ -12,6 +12,7 @@
   Find Case Law
 {% endblock title %}
 {% block precontent %}
+  {% include "includes/survey_banner.html" %}
```

Comment:
I was going to ask if there is a higher level template that is shared between judgment/detail and pages.home that we could put the banner in so we could avoid including it twice, but looking at the screenshots, we have them in different parts of the page and so it would require some sort of refactor to the base template that probably isn't worth it/ may not make sense.